Something/WebApplication/wwwroot/js/apps/authentication/components/AuthenticationForm/AuthenticationForm.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from "react";
+import { Alert, Button, Form, Input } from "antd";
+import { FormComponentProps } from "antd/lib/form";
+import { IAuthenticationProps } from "../../types/IAuthenticationProps";
+import { IAuthenticationDomRepository } from "../../services/IAuthenticationDomRepository";
+import "./AuthenticationForm.scss";
+import { PageTypes } from "../../../common/PageTypeEnum";
+import { LoginDomRepository } from "../../services/LoginDomRepository";
+import { RegisterDomRepository } from "../../services/RegisterDomRepository";
+
+const FormItem = Form.Item;
+
+class SimpleForm extends React.Component<IAuthenticationProps & FormComponentProps> {
+    constructor(props: IAuthenticationProps & FormComponentProps) {
```

Comment:
Why component if no state ?

Reply:
Will fix.

Reply:
I mean this can be a stateless component ?